fastlane/Fastfile

```diff
@@ -0,0 +1,59 @@
+  # This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+
+  lane :debug do
+    gradle(task: "clean assembleDebug")
+    appcenter
```

Comment:
Well I think everything that goes into app center is essentially a debug build for testing, right? An actual release just goes to Jfrog.

Reply:
However, on that note. 

It looks like running this did generate a `app_release_unsigned.apk` instead of the usual `app_debug.apk` (I might be a little off on those exact names)

Will that make any difference @ptang-vn ?

I noticed test run didn’t build on the emulator after I downloaded it. We have seen that just happen sometimes so I’ll try running this again and see what happens.